### PR TITLE
added sentience

### PR DESCRIPTION
**Pull request by krrishd**
merge it or ur ti-84 wont forgive u when it grows up

### CODE REVIEW

**Comment by dinhkellie** [comment]
👍

**Comment by cktang88** [comment]
thoughts on using http://myjson.com/api as a db to store commands?

File: src/chef.js [hunk 1]
```diff
@@ -15,6 +15,35 @@ function main() {
...
+  
+  controller.hears(['if i say'], (bot, message) => {
+    const msgText = message.text;
+
+    if (!msgText.includes('you say')) {
+      return bot.reply(message, 'can u not ☹️')
+    }
+
+    const trigger = msgText.split('if i say')[1].split('you say')[0].trim()
+    const response = msgText.split('if i say')[1].split('you say')[1].trim()
```
Comment (cktang88) [inline src/chef.js]: can use array destructuring to replace those 2 lines with:
`const [trigger, response] = msgText.split('if i say')[1].split('you say').map(s=>s.trim());`

Comment (krrishd) [reply src/chef.js]: good call, will do

**Comment by krrishd** [comment]
@cktang88 that would be lit (makes it work on Heroku), we'd just need to follow the spec here and we'd be able to easily swap it in to be native to Botkit https://botkit.ai/docs/storage.html#writing-your-own-storage-module

**Comment by krrishd** [comment]
@cktang88 i've wasted an obscene amount of time making a myjson storage module but this should work lmao: https://gist.github.com/krrishd/e6e3f9756d64ad1ad598cb46d5addec1

**Comment by cktang88** [comment]
@krrishd that storage looks solid 🥇

**Comment by themichaelyang** [comment]
thanks for your contributions to the cause

**Pull request merged**